utils/create_circleci_config.py

```diff
@@ -0,0 +1,371 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import os
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional
+
+import yaml
+
+
+COMMON_ENV_VARIABLES = {"OMP_NUM_THREADS": 1, "TRANSFORMERS_IS_CI": True, "PYTEST_TIMEOUT": 120}
+COMMON_PYTEST_OPTIONS = {"max-worker-restart": 0, "dist": "loadfile", "s": None}
+DEFAULT_DOCKER_IMAGE = [{"image": "cimg/python:3.7.12"}]
+TORCH_SCATTER_INSTALL = "pip install torch-scatter -f https://pytorch-geometric.com/whl/torch-1.12.0+cpu.html"
```

Comment:
Given we have full control here, could this be inferred from the torch version? It would be simpler than remembering to update this when we update the torch version

Reply:
That would require installing torch for the test fetching job which I'd like to avoid since it takes time. We also have to wait for those wheels to be out, so setting them manually (once now instead of 10 times) is probably better.

Reply:
Will setting `TORCH_SCATTER_INSTALL` to 
```python
pip install torch-scatter -f https://pytorch-geometric.com/whl/torch-$(python -c "from torch import version; print(version.__version__.split('+')[0])")+cpu.html
```
work well here?